strings/names/cats.txt

```diff
@@ -2,13 +2,15 @@ names@=
 	Gary@,Mittens@,Mr. Jingles@,Rex@,Jasmine@,Litterbox
 	Reginald@,Poosycat@,Dr. Purrsworthy@,Lt. Scratches@,Michael Catson
 	Fluffy@,Mr. Purrfect@,Lord Furstooth@,Lion-O@,Johnathan@,Gary Catglitter
-	Chat de Gaulle@,Ratbag
+	Chat de Gaulle@,Ratbag@,Mitts@,Mr. Fluffington@,Meowxie@,Fluffy The Cat
```

Comment:
```suggestion
	Chat de Gaulle@,Ratbag@,Mitts@,Mr. Fluffington@,Meowxie
```
we already had `fluffy` above and I didn't notice when suggesting that one 😔

Reply:
Fixed it.